source/_code.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="container">
+  <button aria-controls="<%= name %>-snippets"
+          aria-expanded="false"
```

Comment:
Correct. However, that is just the case with screen readers. It’s possible other assistive technologies treat that differently. And if we also treat CSS as a progressive enhancement, the button will be present whether it is functioning or not. It would be even more bullet-proof by starting with no button, and injecting one as technologies become available.

I acknowledge this is splitting hairs, but thought I’d mention! I’ll leave it up to you whether or not to take this approach.

Here is what this looks like with no CSS or JavaScript:

![screen shot 2017-01-20 at 11 58 58](https://cloud.githubusercontent.com/assets/903327/22157969/edae9dce-df07-11e6-887a-4c9ea7549941.png)